run/helm/snap/templates/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: snap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  template:
+    metadata:
+      name: snap
+      labels:
+        daemon: snapteld
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
```

Comment:
Would a simple call to Snap API be sufficient in case of readinessProbe? Or do we want to check also if all tasks are in `Running` state?
Same with livenessProbe - we can call Snap API to check if `snapteld` is running. However, it will not discover task failure. And adding checking tasks states is tricky too. I.e. If one task out of 5 gets disabled do we want to restart whole container with the rest of tasks running?